src/clr-angular/data/datagrid/animation-hack/row-expand-animation.spec.ts

```diff
@@ -45,6 +45,19 @@ export default function(): void {
       this.expand.expanded = true;
       expect(this.clarityElement.style.height).toBeTruthy();
     });
+    it(
+      'successive triggering of expand and collapse should not throw an error during `animate()`',
+      fakeAsync(() => {
+        expect(() => {
+          this.expand.expanded = false;
+          this.expand.expanded = true;
```

Comment:
This doesn't really change the expanded state and cause toggling, since you're inside of a fakeAsync. I'm still not sure how to reproduce this issue clearly, but this test isn't really doing anything with animations to address your core issue. I'm still trying to get an isolated demo here.

Reply:
slow down the CPU profile and the animations will fail if a user rapidly clicks on the row expansion icon
Sometimes toggling row expansion on a row that has a radio button, but is in the unselected state can cause the problem ( in a real app, have seen it )

The test I wrote may not even work on the pipeline, because the original Clarity lead developer stated the fact 2 years ago in the test file. It was never corrected.

If you leave my test intact and revert the small bit of code changes in the PR, what happens if you perform said test locally? share any results

Reply:
This test is inside of a non-running test suite, so let's just revert it.